nautobot_golden_config/jobs.py

```diff
@@ -150,7 +151,11 @@ def run(self, data, commit):
 
         # Commit / Push each repo after job is completed.
         for intended_repo in intended_repos:
-            self.log_debug(f"Push new intended configs to repo {intended_repo.url}.")
+            # Remove user and token from url (https://user:password@service.example/repo.git)
+            parsed_repo_url = urlparse(intended_repo.url)
+            self.log_debug(
+                f"Push new intended configs to repo {parsed_repo_url.scheme}//{parsed_repo_url.hostname}{parsed_repo_url.path}."
```

Comment:
```suggestion
                f"Push new intended configs to repo {parsed_repo_url.scheme}//{parsed_repo_url.netloc}{parsed_repo_url.path}."
```

Would this be better to account for port?

```
>>> urlparse("https://my-gitlab.com:8080/any").netloc
'my-gitlab.com:8080'
>>> urlparse("https://my-gitlab.com:8080/any").hostname
'my-gitlab.com'
>>>
```